### PR TITLE
ananicy_cpp-rules -> ananicy-rules-cachyos_git: move to gitOverride

### PR DESCRIPTION
**Pull request by JohnRTitor**
### :fish: What?

1. It's packaged in nixpkgs, so we could use an override.

### :fishing_pole_and_fish: Why?

- (1) This reduces the tarball size?

### :whale: Extras

- I like chocolate cake.

### Questions:

Are automatic update scripts affected by this? Let me know if I have to change something.

### CODE REVIEW

**Comment by JohnRTitor** [comment]
Should be ready to merge, I tried to move smaller packages to gitOverride before trying the big giant :)

File: overlays/default.nix [hunk 1]
```diff
@@ -83,7 +83,7 @@ in
 
   alacritty_git = callOverride ../pkgs/alacritty-git { };
 
-  ananicy-cpp-rules = final.callPackage ../pkgs/ananicy-cpp-rules { };
+  ananicy-cpp-rules = callOverride ../pkgs/ananicy-cpp-rules { };
```
Comment (PedroHLC) [inline overlays/default.nix:86]: Suffix it with `_git` here, and to be more explicit, it should be named `ananicy-rules-cachyos_git`. (`nyxKey` must be updated to match, no need to change the directory's name)

**Review by PedroHLC — CHANGES_REQUESTED** [review]

File: overlays/default.nix (hunk 1 above)
Comment (JohnRTitor) [reply overlays/default.nix:86]: Sure, I just wanted to make backwards-compatibility.

Comment (JohnRTitor) [reply overlays/default.nix]: Done, do you want me to maintain an alias?

Comment (PedroHLC) [reply overlays/default.nix]: Nope

**Review by PedroHLC — APPROVED** [review]

**Pull request merged**